util-images/network/netperfbenchmark/pkg/worker/worker.go

```diff
@@ -19,48 +19,67 @@ limitations under the License.
 package worker
 
 import (
-	"bufio"
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
-	"io"
 	"net/http"
-	"os/exec"
+	"os"
 	"strconv"
 	"time"
 
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
 )
 
-// Worker hold data required for facilitating measurement.
+// Worker holds data required for facilitating measurement.
 type Worker struct {
-	stopCh       chan (struct{})
-	parsedResult []float64
-	err          error
-	protocol     string
-	workerDelay  time.Duration
-	startTime    time.Time
+	k8sClient         dynamic.Interface
+	resourceInterface dynamic.ResourceInterface
+	gvr               schema.GroupVersionResource
```

Comment:
let's remove this one (see my comment below)

Reply:
Modified.